driver/src/main/java/org/neo4j/driver/internal/async/ExplicitTransaction.java

```diff
@@ -130,7 +104,7 @@ else if ( state != State.COMMITTED && state != State.ROLLED_BACK )
     {
         if ( state == State.COMMITTED )
         {
-            return completedWithNull();
+            return failedFuture( new ClientException( "Can't commit, transaction has been committed" ) );
```

Comment:
I'm definitely not an English guru, but I would find `Can't commit, because transaction has already been committed.` more descriptive. And it would be nice if we could standardise the error messages like using always `Can't` or `Cannot` in the same context.